src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -207,39 +186,6 @@ describe('Add Bulk Concepts', () => {
     expect(wrapper.state().openResultModal).toBe(false);
   });
 
-  it('can search for and select "testSource ..." in conceptSources', () => {
-    const wrapper = mount(<MemoryRouter>
-      <Provider store={store}>
-        <AddBulkConcepts {...props} />
-      </Provider>
-    </MemoryRouter>);
-    const other = wrapper.find('#otherSourcesOption');
-    other.simulate('click');
-    const input = wrapper.find('#sourceSearch');
-    input.simulate('change', { target: { value: 'testSource' } });
-    input.simulate('keydown', { key: 'ArrowDown' });
```

Comment:
What happens if we leave this test?

Reply:
It will fail because its searching through the component to find an Id/node `#otherSourcesOption`that does not exist hence it will throw an error saying. `This method is only meant to be run on single node. 0 found instead.`